src/models/hooks/goalFieldResponse.js

```diff
@@ -68,6 +68,13 @@ const syncActivityReportGoalFieldResponses = async (sequelize, instance, options
           },
         },
       );
+
+      // Get a list of activity report ids from idsToUpdate.
+      const activityReportIds = idsToUpdate.map((item) => item.activityReportGoal.activityReportId);
+      // We need to update the AR createdAt so we don't pull from outdated local storage.
+      if (activityReportIds.length > 0) {
+        await sequelize.query(`UPDATE "ActivityReports" SET "updatedAt" = '${new Date().toISOString()}' WHERE id IN (${activityReportIds.join(',')})`);
+      }
```

Comment:
@AdamAdHocTeam 

Why was this done this way and not using the model structure?
This is a backend sequelize hook, how does this have anything to do with local storage on the users browser?

Reply:
I think i understand why we need to update the date now, but not why it was done with raw SQL. using raw SQL opens us up for more parsing issue. Dates most specifically.

Reply:
We cannot update the updatedAt column through the model process.

Reply:
@AdamAdHocTeam 
`await ActivityReport.update({ updatedAt }, { where: { id: activityReportIds }, silent: true });`